pkg/api/frontendsettings.go

```diff
@@ -198,9 +198,10 @@ func (hs *HTTPServer) getFrontendSettingsMap(c *models.ReqContext) (map[string]i
 		"unifiedAlerting": map[string]interface{}{
 			"minInterval": hs.Cfg.UnifiedAlerting.MinInterval.String(),
 		},
-		"oauth":       hs.getEnabledOAuthProviders(),
-		"samlEnabled": hs.samlEnabled(),
-		"samlName":    hs.samlName(),
+		"oauth":                   hs.getEnabledOAuthProviders(),
+		"samlEnabled":             hs.samlEnabled(),
+		"samlName":                hs.samlName(),
+		"tokenExpirationDayLimit": hs.Cfg.TokenExpirationDayLimit,
```

Comment:
there's an "auth" map in this section, do you think `tokenExpirationDayLimit` fits there?

Reply:
I'd argue the new option falls under the new section `[service accounts]` with only 1 option. I believe it's ok for the option to be all by itself, in case SA section grows.

WDYT?

Reply:
Fine by me :+1: